### PR TITLE
Added support for guregu null types as a CLI option

### PR DESCRIPTION
**Pull request by snagles**
@Shelnutt2 let me know what you think.

### CODE REVIEW

**Comment by stefannaglee** [comment]
@Shelnutt2 let me know what you think.

File: utils.go [hunk 1]
```diff
@@ -273,3 +281,54 @@ func mysqlTypeToGoType(mysqlType string, nullable bool) string {
...
+		if nullable {
+			return "null.Int"
+		}
+		return "int64"
+	case "varchar":
+		if nullable {
+			return "null.String"
+		}
+		return "string"
+	case "datetime":
```
Comment (Shelnutt2) [inline utils.go:308]: guregu supports null.Time

**Comment by Shelnutt2** [comment]
The CI build failed. Looks like the import should be Shelnutt2 instead of shelnutt2

File: db2struct/main.go [hunk 2]
```diff
@@ -6,10 +6,10 @@ import (
 	"os"
 	"strconv"
 
-	"github.com/shelnutt2/db2struct"
 	goopt "github.com/droundy/goopt"
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/howeyc/gopass"
+	"github.com/shelnutt2/db2struct"
```
Comment (Shelnutt2) [inline db2struct/main.go]: Should be Shelnutt2

**Pull request merged**